python/mxnet/ndarray/numpy/_op.py

```diff
@@ -2363,3 +2363,55 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):  # pylint:
     0.2025
     """
     return _npi.var(a, axis=axis, dtype=dtype, ddof=ddof, keepdims=keepdims, out=out)
+
+
+@set_module('mxnet.ndarray.numpy')
+def ravel(x, out=None):
```

Comment:
In numpy, we have the `order='C'` argument. We need to add that and raise NotImplementedError if the value is other than `C`.